language/bytecode-verifier/src/unused.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! This module contains verification of usage of dependencies for modules
```

Comment:
I would name this module something else and add a bit more description. It is not obvious to me what sort of things is checked in the `unused` bytecode verifier pass

Reply:
Maybe `unused_module_aliases` or `unused_aliases`?

Reply:
We no longer need the `unused` checker.